### DISCUSSION TITLE
AWS Lambda example ?

### DISCUSSION THREAD

**Discussion by tclain-pillar** [hattipjs/hattip#71] [Q&A]
Hi Folks!

I am coming from hono that offers an official lambda adapter.

The hattip ecosystem is really appealing to us, and we would like to use a tool that standardize on Request /Response.

However I don't see any mention of hattip working in lambda.

What is the recommended stance ? is it supported ? Do we have to build a small adapter ourselves ?

Thx

Tim.

**Comment by cyco130** [comment]
Hi!

That's correct, we don't have an AWS adapter yet. I just opened [an issue](https://github.com/hattipjs/hattip/issues/72) for it since it's been requested again recently.

I don't expect it to be a lot of work, I'll try to have a look at the first opportunity.
